omod/src/main/webapp/resources/scripts/jquery-ui/js/openmrsSearch.js

```diff
@@ -951,6 +963,12 @@ function OpenmrsSearch(div, showIncludeVoided, searchHandler, selectionHandler,
 
         _doSelected: function(position, rowData) {
             if(this.options.selectionHandler) {
+        		// Check the browser compatibility and, add an entries to sessionStorage
+            	if(typeof(Storage) !== "undefined") {
+        	        sessionStorage.lastSearchText= $j('#inputNode').val();
+        			sessionStorage.includeVerbose = ($j(verboseCheckBox).is(':checked')) ? "checked" : "unchecked";
```

Comment:
You could possibly store any random text you want, preferably 'true' but i don't you need 2 of them for each state of the checkbox, presence of the single text you choose to use should be enough, if it doesn't exist then the checkbox wasn't checked